Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Global variables
+ARG COMMIT=""
+ARG APP_USER=lyric
+ARG WORKDIR=/usr/src/app
+
+######################
+# Configure base image
+######################
+FROM node:18.16.1-alpine AS base
```

Comment:
It's a new app, let's aim for today's LTS (v20.*)

Reply:
updated to version `20.12.2-alpine` currently the latest